.devcontainer/codfsh-config.yaml

```
@@ -1,9 +1,9 @@
-sushi:
-  min_version: "3.5.0"
-hapi:
-  min_version: "3.0.0"
-  parameters: 
-    jurisdiction: DE
-    locale: de-DE
-    tx: "n/a"
-    debug: true
```

Laut der GitHub Seite kann hier noch "proxy: 192.168.110.10:3128" eingefügt werden.
https://github.com/gematik/codfsh
```suggestion
    debug: true
    proxy: 192.168.110.10:3128
```